modules/parquet/src/parquetjs/reader.ts

```diff
@@ -406,10 +408,42 @@ export class ParquetEnvelopeReader {
     ) as any;
 
     const pagesOffset = Number(colChunk.meta_data?.data_page_offset!);
-    const pagesSize = Number(colChunk.meta_data?.total_compressed_size!);
-    const pagesBuf = await this.read(pagesOffset, pagesSize);
+    let pagesSize = Number(colChunk.meta_data?.total_compressed_size!);
 
-    return await decodeDataPages(pagesBuf, field, compression);
+    if (!colChunk.file_path) {
```

Comment:
Nit: What happens if there is a file path? Do we throw an exception? Do we ignore the column?

Reply:
Yes we throw an exception like: 

`if (colChunk.file_path !== undefined && colChunk.file_path !== null) {
      throw new Error('external references are not supported');
 }`